docs/app/documentation/component-docs/time-picker/examples/time-picker-form-example.component.html

```diff
@@ -0,0 +1,9 @@
+<form [formGroup]="customForm">
+    <fd-time-picker formControlName="time"></fd-time-picker>
+</form>
+
+Selected Time: <span *ngIf="customForm.controls.time.value">
```

Comment:
instead of having an ngIf, pretty sure you could just go like 

`customForm.controls.time.value?.hour` or something, I forget the exact syntax, but yeah. The question mark basically does a nullcheck

Reply:
Looks like this is a CoffeeScript feature, not part of TypeScript yet

https://github.com/Microsoft/TypeScript/issues/16

Reply:
I literally use it in another part of this app lol

Reply:
@mikerodonnell89 The ticket you linked is number 16, it was probably added long ago but never closed.